print ("input.py")
raise Exception("soon exception")
